src/populate-persistence.ts

```diff
@@ -0,0 +1,342 @@
+import faker from "faker/locale/it";
+import { range } from "fp-ts/lib/Array";
+import fs from "fs";
+import _ from "lodash";
+import { CreatedMessageWithContent } from "../generated/definitions/backend/CreatedMessageWithContent";
+import { CreatedMessageWithContentAndAttachments } from "../generated/definitions/backend/CreatedMessageWithContentAndAttachments";
+import { EUCovidCert } from "../generated/definitions/backend/EUCovidCert";
+import { MessageAttachment } from "../generated/definitions/backend/MessageAttachment";
+import { MessageSubject } from "../generated/definitions/backend/MessageSubject";
+import { PrescriptionData } from "../generated/definitions/backend/PrescriptionData";
+import { ioDevServerConfig } from "./config";
+import {
+  createMessage,
+  getMvlAttachments,
+  withContent,
+  withDueDate,
+  withLegalContent,
+  withPaymentData
+} from "./payloads/message";
+import MessagesDB from "./persistence/messages";
+import { eucovidCertAuthResponses } from "./routers/features/eu_covid_cert";
+import { services } from "./routers/service";
+import { IoDevServerConfig } from "./types/config";
+import {
+  frontMatter1CTABonusBpd,
+  frontMatter1CTABonusBpdIban,
+  frontMatter1CTABonusCgn,
+  frontMatter2CTA2,
+  frontMatterBonusVacanze,
+  messageMarkdown
+} from "./utils/variables";
+
+const getRandomServiceId = (): string => {
+  if (services.length === 0) {
+    throw new Error(
+      "to create messages, at least one sender service must exist!"
+    );
+  }
+  return faker.random.arrayElement(services).service_id;
+};
```

Comment:
```suggestion
const getServiceId = (): string => {
  if (services.length === 0) {
    throw new Error(
      "to create messages, at least one sender service must exist!"
    );
  }
  return getRandomValue(
    services[0].service_id,
    faker.random.arrayElement(services).service_id,
    "messages"
  );
};
```
to make the global config more efficient, could be a good idea to return a random `serviceId` only when the randomness is enabled for this specific section (`messages`)?

Reply:
No opinions from my side: is a testing tool and I see no value in performance tuning.
Your call 😃

Reply:
What I meant with `Efficient` is not about performance but to make the dev-server deterministic (or with a certain degree of randomness) by configuration. 

As you can see each sections of the configuration (messages, profile, services etc..) has an optional flag `allowRandomValue` . This is useful in dev-environment (or in automatic e2e tests) when you want to load always the same data and avoid randomness.

Reply:
Got it. I was misled by _efficient_.